src/SDKs/ContainerRegistry/Management.ContainerRegistry/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Container Registry Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Container Registry.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
@northtyphoon if this is your next 3.0 release, I would suggest you start your versioning from 2.8.0
This will allow and give you room to do bug fixes for your 2.0 stable release.
So in that case your nuget version should be 2.8.0-preview and Assembly Version = 2.0.0.0 and AssemblyFileVersion=2.8.0.0

Reply:
Thank. Update

---
In reply to: [207694516](https://github.com/Azure/azure-sdk-for-net/pull/4630#discussion_r207694516) [](ancestors = 207694516)